src/components/quizPage/AnswersContainer.styles.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+interface AnswersCountProp {
+    count: number
+}
+
+const AnswersGroup = styled.ul<AnswersCountProp>`
+  display: grid;
+  gap: 15px;
+  grid-template-rows: ${({ count }) => (count >2 ? 'repeat(2, 1fr)' : 'repeat(1, 1fr)')};
```

Comment:
```suggestion
  grid-template-rows: ${({ count }) => `repeat(${count}, 1fr)`;
```
yes, you'll have to pass a count every time but AnswersGroup will be more flexible